anvio/docs/artifacts/functions.md

```diff
@@ -0,0 +1,11 @@
+This artifact **contains information about the functions of the contigs in your %(contigs-db)s.**
+
+It is less a separate file, and more an extension of your %(contigs-db)s that contains this information. 
```

Comment:
Annotated functions are part of the contigs database; specifically, they go in the gene functions table of the database. So I wouldn't describe them as being in a file, per se; that might confuse some people into thinking they could find said file somewhere on their computer, but they cannot because this information is internal to the database. 

This is definitely one of the more abstract artifacts :)

Perhaps a better way to describe functions is "a table in your contigs database that contains functional annotations for genes".

Reply:
Oh sorry! This was one of the first ones I wrote, when I thought that all of the artifacts were file types. I'll fix that now.